examples/extended/create.sh

```diff
@@ -5,7 +5,8 @@ kubectl.sh create -f existing_job.yaml
 
 kubectl.sh create -f ../../manifests/appcontroller.yaml
 
-sleep 10
+#wait for appcontroller pod creation
+sleep 20
```

Comment:
Yeah, as I said in previous comment, we should probably query k8s api for required resources.